src/App.css

```diff
@@ -172,3 +177,27 @@ a.home-button svg {
   width: 80%;
   height: 300px;
 }
+
+/* Dictionary Styling */
+.buttons {
```

Comment:
if it's applied to a specific component style, then i think it's better to use tailwind, but putting all of them in App.css